### PR TITLE
Update phone number assignment cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
We are changing the way phone numbers are assigned to Microsoft Teams users and resources. This PR adds deprecation notice to the existing cmdlets and points readers to using the new cmdlets.

Also adds a new example to set-csphonenumberassignment.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Provide the topic introduction here.
 
+> [!NOTE]
+> This cmdlet will be deprecated.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md:19]:
```suggestion
**Note**: This cmdlet will be deprecated.
```

File: skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md [hunk 2]
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Updates an application instance in Azure Active Directory. 
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md:19]:
```suggestion
**Note**:  This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```

File: skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md [hunk 3]
```diff
@@ -6,15 +6,18 @@ title: Set-CsOnlineVoiceApplicationInstance
...
+ms.author: jenstr
 ---
 
 # Set-CsOnlineVoiceApplicationInstance
 
 ## SYNOPSIS
 The `Set-CsOnlineVoiceApplicationInstance` modifies an application instance in Azure Active Directory.
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md:19]:
```suggestion
**Note**: This cmdlet will be deprecated. Start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) instead.

```

File: skype/skype-ps/skype/Set-CsOnlineVoiceUser.md [hunk 4]
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Set-CsOnlineVoiceUser` cmdlet to set the PSTN specific parameters (like telephone numbers and emergency response locations.)
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineVoiceUser.md:19]:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 5]
```diff
@@ -17,6 +17,9 @@ Modifies Skype for Business properties for an existing user account.
 Properties can be modified only for accounts that have been enabled for use with Skype for Business.
 This cmdlet was introduced in Lync Server 2010.
 
+> [!NOTE]
+> Using this cmdlet for Microsoft Teams users will be deprecated. For details, please see individual parameters below.
+> Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:22]:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 6]
```diff
@@ -158,6 +161,8 @@ Accept wildcard characters: False
 Indicates whether the user has been enabled for Enterprise Voice, which is the Microsoft implementation of Voice over Internet Protocol (VoIP).
 With Enterprise Voice, users can make telephone calls using the Internet rather than using the standard telephone network.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:165]:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 7]
```diff
@@ -176,6 +181,9 @@ Accept wildcard characters: False
 When set to True, enables a user's voice mail calls to be routed to a hosted version of Microsoft Exchange Server.
 In addition, setting this option to True enables Skype for Business users to directly place a call to another user's voice mail.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated and it is not required anymore to set it for Microsoft Teams users.
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:186]:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 8]
```diff
@@ -407,6 +415,9 @@ Depending on your setup, those two numbers could actually be different.
 For example, in some organizations dialing 1-425-555-1297 routes your call to an Exchange Auto Attendant.
 Conversely, dialing just the extension (51297) or using Skype for Business to dial the number 1-425-555-1297 extension 51297 will route your call directly to the user.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:419]:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

Comment (JensTrier) [reply skype/skype-ps/skype/Set-CsUser.md:417]: @JohanFreelancer9 the cmdlet Set-CsUser itself will stay, but not for Teams. So it's really the parameters that only will be deprecated

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

@JohanFreelancer9 please see comment above on set-csuser. That cmdlet stays, since it is used for SfB On-prem. It's only the parameters used for Teams that will be deprecated. That's why I had the note on the parameters

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@JensTrier, thank you for the clarification. I have changed my suggestions accordingly. Could you please commit the remaining three changes if you agree with them? Many thanks.

**Comment by JensTrier** [comment]
> @JensTrier, thank you for the clarification. I have changed my suggestions accordingly. Could you please commit the remaining three changes if you agree with them? Many thanks.

Thanks, I did your changes. I also changed the note on top of Set-CsUser. Please have a look.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @JensTrier.

**Comment by Scifire** [comment]
@JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.

I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.

**Comment by JensTrier** [comment]
> @JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.
> 
> I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.

Thanks for the feedback! I added this note to Set-CsUser for the HostedVoiceMail parameter: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.

Don't you see that?

**Comment by Scifire** [comment]
> > @JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.
> > I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.
> 
> Thanks for the feedback! I added this note to Set-CsUser for the HostedVoiceMail parameter: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.
> 
> Don't you see that?

Ahh sorry, my mistake. Didn´t checked the description of the parameter in the PR. Okay then everything should be explained and fine. Thanks :)

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**